apis/python/tests/test_tiledbvcf.py

```diff
@@ -1106,6 +1106,7 @@ def test_ingest_mode_merged(tmp_path):
     assert ds.count(regions=["chrX:9032893-9032893"]) == 0
 
 
+@pytest.mark.skipif(shutil.which("bcftools") is None, reason="no bcftools")
```

Comment:
> Can we make this more specific to something like: windows+ `"CI" in os.environ`?

I agree; skipping the tests that rely on BCFTools to prepare inputs has proven necessary in the past; we need it to be bulletproof on at least one platform.

Reply:
Good call. I just force pushed an update so that the tests are only skipped if all 3 of the following conditions are true:

* env var `CI` is `true` (set by [default](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) on GitHub Actions)
* The OS is Windows (I purposefully used `platform.system()` here to match the code in this test file)
* There is no bcftools installed (if someone goes through the trouble to install bcftools on Windows, we should run the tests!)

The only downside is that the test is going to fail for local builds that use the conda env I provided. Currently I suspect David and I are the only ones locally building from source on Windows, so this is a minor inconvenience.